functions/Get-DbaBackupDevice.ps1

```diff
@@ -41,7 +44,8 @@ Returns all Backup Devices for the local and sql2016 SQL Server instances
 	Param (
 		[parameter(Position = 0, Mandatory = $true, ValueFromPipeline = $True)]
 		[DbaInstanceParameter]$SqlInstance,
-		[System.Management.Automation.PSCredential]$SqlCredential
+		[System.Management.Automation.PSCredential]$SqlCredential,
```

Comment:
This should be set to `[PSCredential][System.Management.Automation.CredentialAttribute()]`.